src/frontend/src/services/api/__tests__/controller.test.ts

```diff
@@ -30,16 +30,23 @@ describe('getApi', () => {
     expect(api).toBe(RemoteMockApi);
   });
 
-  it('should return LocalMockApi when REACT_APP_API_MODE is "LOCALMOCK"', () => {
-    process.env.REACT_APP_API_MODE = 'LOCALMOCK';
+  it('should return LocalMockApi when VITE_APP_API_MODE is "LOCALMOCK"', () => {
     const result = getApi();
     expect(result).toEqual(LocalMockApi);
   });
 
-  it('should return RemoteMockApi when REACT_APP_API_MODE is "REMOTEMOCK"', () => {
-    process.env.REACT_APP_API_MODE = 'REMOTEMOCK';
+  it('should return RemoteMockApi when VITE_APP_API_MODE is "REMOTEMOCK"', () => {
+    jest.resetModules();
+    jest.mock('../../../config/config', () => ({
+      config: {
+        VITE_APP_API_MODE: 'REMOTEMOCK',
+      },
+    }));
```

Comment:
Does this impact the global mock in `jest-setup.ts`?

Reply:
Yes, the global mock set it as 'LOCALMOCK' however in this instance we needed to test it as 'REMOTEMOCK' - after a few work arounds this is the only I got round it was calling the api again after mocking the config in the test